purchase_stock_picking_invoice_link/models/stock_move.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.multi
+    def _get_moves(self):
```

Comment:
This method name is too general as it can exist in another module.

Maybe _get_purchase_invoice_link_moves()